hack/upgrade-test-index-image.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+# Usage:
+# make upgrade-test-index-image
+#
+# Use Openshift-CI "optional-operators-ci-*" workflow to:
+# - Build an internal index image based off the index image at:
```

Comment:
based off => based of

Reply:
It should be either _based off (US) of_ or _based on_ (UK)
https://english.stackexchange.com/questions/15081/based-on-instead-of-based-off-of